AN00209_xCORE-200_DSP_Library/app_window_post_fft/Makefile

```diff
@@ -1,7 +1,18 @@
+BUILD_FLAGS = -O2 -g -report
+
+XCOREAI ?= 0
+ifeq ($(XCOREAI),0)
```

Comment:
Same comment as app_adaptive Makefile.

Reply:
This has been fixed by https://github.com/xmos/lib_dsp/pull/193